### ISSUE TITLE
How do you circle crop Profile ImageIcon and disable arrow icon on accountheader?

### ISSUE THREAD

**Issue by michellemmy** [mikepenz/MaterialDrawer#2467]
Hi, first of all, I'm quite new to android programming and might need a little assistance here.
By the way, this library is so awesomely complete and I'm just wondering how long did it took you to build this. Like wow! Amazing!

Now, the little issue I have are :
1. Whenever I clicked on to the hamburger icon, the drawer would display account header and the remaining list of account *which I don't want it to be display*
2. The Profile Icon is a uri format and not properly center crop to fit the circle container, it leaves a blank area there as you see.

For the first issue, I was wondering if the drawer would display the current account header and the DrawerBuilder (list of menus) without me clicking the arrow button to hide if there are more than 1 profile.

I have searched and googled everywhere and have not found any proper solution to my problem. Thanks a lot, and sorry for my beginner English 😄 

<img width="318" alt="Screen Shot 2019-07-12 at 15 47 12" src="https://user-images.githubusercontent.com/52818822/61115719-44b50900-a4bd-11e9-87a4-467f06d34ff1.png">

**Comment by mikepenz** [comment]
Hi. Thank you very much. I hope it helps you with your development.

For your first issue please see the following:
https://github.com/mikepenz/MaterialDrawer/blob/develop/FAQ/accountheader_single_profile_without_dropdown.md

For the second issue:
The quickest solution would be that you copy over the xml file of the header to your project (keep the name) and change the scaleType of the imageView to be a different one meeting your requirements:
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/res/layout/material_drawer_header.xml#L12
The better and cleaner solution would be that you set a square image with your image loading library (glide or picasso have some different APIs to make an image you load be square, please refer to those libraries documentation accordingly)

**Comment by michellemmy** [comment]
Hi, that was a fast respond! 
I have followed your instruction and solved the second issue. 
Thank you so much!

For the first issue, I did add the code on the AccountHeaderBuilder,
and on the first run, yes it did just right. Hide those selection profile list and showing the menus on the first time minidrawer.
<img width="296" alt="Screen Shot 2019-07-12 at 16 28 42" src="https://user-images.githubusercontent.com/52818822/61118083-31586c80-a4c2-11e9-9ac9-543465a3d776.png">

But if I clicked on this area here *yellow dot*
<img width="64" alt="Screen Shot 2019-07-12 at 16 29 47" src="https://user-images.githubusercontent.com/52818822/61118243-7f6d7000-a4c2-11e9-9b91-a61f87d6a866.png">



It still shows this and not showing the menus again if I drag the drawer to the right side.
Menus will show if I clicked one of the menu and redrag the drawer again.
<img width="321" alt="Screen Shot 2019-07-12 at 16 24 00" src="https://user-images.githubusercontent.com/52818822/61117805-919ade80-a4c1-11e9-8b32-72883635c922.png">

Quite annoying right? 😆
